apps/range-voting/app/components/Panels/VotePanelContent.js

```diff
@@ -261,7 +259,10 @@ class VotePanelContent extends React.Component {
             <h2>
               <Label>Your voting tokens</Label>
             </h2>
-            {BigNumber(this.state.userBalance).toString()}
+            {BigNumber(this.state.userBalance)
+              .div(BigNumber(this.state.decimals))
+              .dp(3)
+              .toString()}
```

Comment:
There is no clear advantage on having these values into the component state, instead I suggest to calculate the balance as it should be shown on the view and pass it as a prop from the background worker, should that make sense? We can let this way for a while but this seems like technical debt